tools/ci-build/scripts/check-rust-runtimes-and-tools

```diff
@@ -23,6 +23,7 @@ do
     cargo clippy --all-features
     cargo test --all-features
     cargo doc --no-deps --document-private-items --all-features
+    cargo minimal-versions check --all-features
```

Comment:
Not blocking for this PR, but we need to split `check-rust-runtimes-and-tools` up since its now our long pole.